package.json

```diff
@@ -29,38 +29,38 @@
     "url": "https://github.com/algonauti/ember-g-recaptcha/issues"
   },
   "dependencies": {
-    "ember-cli-babel": "^6.16.0"
+    "ember-cli-babel": "^7.11.1",
```

Comment:
this change means that we now need a minimum Ember CLI version of 2.13, which should probably also be considered a breaking change

Reply:
The disclaimer "This addon only supports Ember 3 or above" has been in the Readme since version 1.0.0